docs/computing/jobs/partitions.md

```diff
@@ -46,7 +56,7 @@ that your job may share the node with other jobs.
     [data analytics][lumid] part of LUMI and have 4TB of memory. Another 
     difference is that the CPUs architecture of these nodes is Zen2.
 
-## Getting information about partitions
+## Getting information about Slurm partitions
 
 A list of the available partitions can be obtained using the `sinfo` command.
```

Comment:
Would it be good to mention this already earlier in the text / in the beginning of the chapter?

Reply:
Good idea! I have more this line about `sinfo` to the top of the page.